iot/api-client/mqtt_example/cloudiot_mqtt_example.py

```diff
@@ -206,7 +213,7 @@ def parse_command_line_args():
     parser.add_argument(
             '--mqtt_bridge_port',
             choices=(8883, 443),
-            default=8883,
+            default=443,
```

Comment:
shouldn't 8883 be the default (and 443 be backup for firewall issues/testing)?

Reply:
It'd appeared that the tests were failing because of the default port, updating test / fixing.

Reply:
I'm seeing the issue again now.